frontend/src/pages/article/components/article-details/article-details.tsx

```diff
@@ -49,6 +57,13 @@ const ArticleDetails: FC<Properties> = ({
             rating
           </li>
         </ul>
+        {!isArticleOwner && !isShared && (
+          <Button
+            className={styles.followButton}
+            label={isFollowed ? 'Unfollow' : 'Follow'}
+            onClick={onFollow}
```

Comment:
Maybe we should put 'Unfollow' and 'Follow' in an enum?

Reply:
[979f92c](https://github.com/BinaryStudioAcademy/bsa-2023-writorium/pull/280/commits/979f92c0d9db98b3d536939a605cbd07d58e31f4)